its/src/test/java/com/sonar/it/scanner/msbuild/sonarqube/BaseDirTest.java

```diff
@@ -88,7 +88,7 @@ void whenMajorityOfProjectsIsOnSameDrive_AnalysisSucceeds() throws IOException {
   }
 
   @Test
-  void testAzureFunctions_WithWrongBaseDirectory_AnalysisSucceeds() throws IOException {
+  void azureFunctions_WithWrongBaseDirectory_AnalysisSucceeds() throws IOException {
```

Comment:
Letter after the underscore is sometimes upper case and sometimes lower case. Keep it consistent within this PR